packages/next/src/server/base-server.ts

```diff
@@ -3053,8 +3054,7 @@ export default abstract class Server<
             routeModule?.definition.kind === RouteKind.APP_PAGE &&
             !isServerAction
           ) {
-            req.headers[RSC_HEADER] = '1'
-            req.headers[NEXT_ROUTER_PREFETCH_HEADER] = '1'
+            req.headers[NEXT_DEV_WARMUP_HEADER] = '1'
```

Comment:
if we're not running the dev server, should we make sure this header deleted and not respected by the render pathway?

Reply:
Yeah though I'm thinking maybe we don't even use a header and just pass an arg through. It's not like this can/should be set from middleware. It only used headers before I think because we were set up to already act on them but this is now a totally novel render type

Reply:
Updated to just use a renderOpt